pkg/splunk/enterprise/configuration.go

```diff
@@ -1041,15 +1041,25 @@ func validateRemoteVolumeSpec(volList []enterpriseApi.VolumeSpec, isAppFramework
 		if isAppFramework {
 			if volume.Type == "" {
 				return fmt.Errorf("Remote volume Type is missing")
+			} else if volume.Type != "s3" {
+				return fmt.Errorf("Remote volume type is invalid. Only storageType=s3 is supported")
 			}
+
 			if volume.Provider == "" {
 				return fmt.Errorf("S3 Provider is missing")
+			} else if !isValidProvider(volume.Provider) {
+				return fmt.Errorf("S3 Provider is invalid")
```

Comment:
Nit: check in line 1048 can also become part of isValidProvider(). all in one place.